SU-HSDO/suhumsci
# HSD8-659 Adding bottom margin for group blocks to space them better

# READY FOR REVIEW

# Summary
- Added bottom margin to group blocks

# Needed By (Date)
- no rush

# Urgency
- low

# Steps to Test
1. sync any site
1. create a basic page
1. edit the layout for that page
1. add 2 group blocks in a sidebar or someplace
1. view the page after saving and ensure the groups have space between them.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
